packages/react-tabs/src/components/TabList/useTabListStyles.ts

```diff
@@ -16,10 +16,22 @@ const useStyles = makeStyles({
   root: {
     display: 'flex',
     flexDirection: 'row',
+    flexWrap: 'nowrap',
+    flexShrink: 0,
     position: 'relative',
+    ...shorthands.overflow('hidden'),
+  },
+  horizontal: {
+    alignItems: 'center',
+    flexDirection: 'row',
+    minWidth: '0',
+    width: '100%',
   },
   vertical: {
     flexDirection: 'column',
+    alignItems: 'flex-start',
```

Comment:
Should this be `align-items: stretch`? That would kinda make more sense to me. Was this supposed to be `justify-content: flex-start`? (kinda same comment on horizontal)

This is probably more obviously relevant if some tabs have e.g. secondary content, or a max-width + text wrap

Reply:
flex-box in column mode has align-items affects the horizontal position of items and justify-items the vertical position.  Per the figma, the tabs are aligned left in vertical and aligned center in horizontal.

Reply:
I think I'm still kinda confused -- shouldn't all horizontally-aligned tabs have the same height, and vertically-aligned tabs have the same width? The latter is already going to be the case anyway, so I don't think `align-items` is doing anything here. For horizontal tabs, right now it's creating this experience when one tab has two lines of text:

![screenshot of tablist with one tab that has two lines of text, showing different heights between tab buttons](https://user-images.githubusercontent.com/3819570/164567466-b150ea6a-339d-44dc-8072-94ae72c2ef58.png)

Reply:
Should be better now.
<img width="685" alt="image" src="https://user-images.githubusercontent.com/28762486/164794545-48fc1f69-4910-4176-b093-24771f362568.png">